### ISSUE TITLE
comms console screen is layered above paper from messages

### ISSUE THREAD

**Issue by factoryman942** [tgstation/tgstation#40048]
> There are serious limitations to our ability to do this (they get layered on top of things that should be on top of them) but with computer screens and apcs at least they shouldn't be an issue.

**Comment by ShizCalev** [comment]
https://github.com/tgstation/tgstation/blob/d82cdaa701cf52170bcbf0b31af6e4df92ae44f5/code/game/machinery/computer/_computer.dm#L58-L64

**Comment by SpaceManiac** [comment]
@ninjanomnom

**Comment by AlexTheSysop42** [comment]
Very likely also happens for supply console manifests but probably not noticed because no one really cares about them

**Comment by ninjanomnom** [comment]
There's no way to actually solve this. At best the issue can be dodged by shifting the paper off the screen.

**Comment by kriskog** [comment]
Yeah, happens to all the consoles that have stuff popping out of them. You will also see the issue when removing disks, so rnd, genetics, cloning.

**Comment by SpaceManiac** [comment]
ACPs are layered above the fog of war
![image](https://user-images.githubusercontent.com/222630/45000045-75abdf80-af76-11e8-8583-7d0e440d3b8e.png)
